testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ApplicationsTest.java

```diff
@@ -88,7 +88,7 @@ public void applicationListTest() throws Exception {
         assertFalse(applications.isEmpty());
         Map<String, ApplicationsPage.ClientRepresentation> apps = applications.stream().collect(Collectors.toMap(x -> x.getClientId(), x -> x));
         assertThat(apps.keySet(), containsInAnyOrder("always-display-client", "account-console"));
-        assertClientRep(apps.get("account-console"), "Account Console", false, true, "/realms/test/account/", false);
+        assertClientRep(apps.get("account-console"), "Account Console", false, true, "https://localhost:8543/auth/realms/test/account/", false);
```

Comment:
Please don't use hardcoded localhost. If we would need to run the tests e.g. with a different host (and we do need that now in some cases to properly test cookies) or without SSL, this won't work.

Something like:
```suggestion
        assertClientRep(apps.get("account-console"), "Account Console", false, true, getAuthServerRoot() + "/realms/test/account/", false);
```

This concerns other lines as well.